README.md

```diff
@@ -179,6 +179,8 @@ aarch64
 
 For full configuration documentation, see [config.md](./docs/config.md).
 
+For tips on getting a rootfs, see [rotfs.md](./docs/rootfs.md).
```

Comment:
```suggestion
For tips on getting a rootfs, see [rootfs.md](./docs/rootfs.md).
```

Reply:
Maybe better to word it like:

"For tips on getting a rootfs that is different from the host, see ..."

So it doesn't confuse folks into thinking they _need_ a fresh rootfs

Reply:
Good point